Source/Core/DolphinQt/Config/BBAConfigWidget.cpp

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt/Config/BBAConfigWidget.h"
+
+#include <QDialogButtonBox>
+#include <QFormLayout>
+#include <QHBoxLayout>
+#include <QLabel>
+#include <QLineEdit>
+#include <QMessageBox>
+#include <QSpinBox>
+#include <QToolButton>
+#include <QVBoxLayout>
+
+#include "Common/Network.h"
+
+BBAConfigWidget::BBAConfigWidget(QWidget* parent) : QDialog(parent)
```

Comment:
I wonder if this dialog should be modal? Or is it modal by default?

Reply:
GameCubePane only shows it as modal as it using .exec() on it